molecule/default/verify.yml

```
@@ -0,0 +1,15 @@
+---
+- name: Verify
+  hosts: all
+  gather_facts: true
+  vars:
+    # load configurations references
+    os_config_ref: "{{ lookup('ansible.builtin.template', '../../defaults/os_config_ref.yml') | from_yaml }}"
+    php_config_ref: "{{ lookup('ansible.builtin.template', '../../defaults/php_config_ref.yml') | from_yaml }}"
+  vars_files:
+    - ../../defaults/main.yml
+
+  tasks:
+    - name: "Include test_install_nextcloud"
+      ansible.legacy.include_role:
+        name: "test_install_nextcloud"
```

> Here should be `ansible.builtin.import_role`

here's the suggestion for direct commit if you wish

```suggestion
    - name: "Include test_install_nextcloud"
      ansible.builtin.import_role:
        name: "test_install_nextcloud"
```